packages/reference/src/components/LinkActions/LinkEntityActions.tsx

```diff
@@ -14,8 +14,8 @@ type LinkEntityActionsProps = {
   canCreateEntity: boolean;
   canLinkEntity: boolean;
   validations: ReferenceValidations;
-  onCreate: (id: string) => void;
-  onLink: (ids: string[]) => void;
+  onCreate: ((id: string) => void) | ((id: string, index?: number) => void);
+  onLink: ((ids: string[]) => void) | ((ids: string[], index?: number) => void);
```

Comment:
Do we need an overload type here? 

```suggestion
  onLink: (ids: string[], index?: number) => void;
```

should be okay, or not?

Reply:
I think I saw an error come up around this when this changeset was in development, but I changed it locally and I don't see it now, so as long as CI is okay with it we can do it this way :)

Reply:
Updated in https://github.com/contentful/field-editors/pull/442/commits/5131e7501b03d3f6964b40738161125da639f2f9.